tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -95,7 +95,7 @@ create_args+=("--image=${INSTANCE_IMAGE:-ssm:/aws/service/canonical/ubuntu/serve
 create_args+=("--set spec.etcdClusters[0].manager.listenMetricsURLs=http://localhost:2382")
 create_args+=("--set spec.etcdClusters[0].manager.env=ETCD_QUOTA_BACKEND_BYTES=8589934592")
 create_args+=("--set spec.etcdClusters[1].manager.env=ETCD_QUOTA_BACKEND_BYTES=8589934592")
-create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=5")
+create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=30")
```

Comment:
I think this should be enough for now. There's no additional speed going further.
```suggestion
create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=10")
```

Reply:
@hakman did you notice client-side rate liming when you were testing out with `20` ?

Reply:
no, but i also did not notice any visible gains